### PR TITLE
aggregateCursor method

### PR DESCRIPTION
**Pull request by bsouthga**
Adds aggregation method returning a cursor, mirroring `promised-mongo`

### CODE REVIEW

File: lib/classes/TemporalCollection.js [hunk 1]
```diff
@@ -101,7 +102,7 @@ export default class TemporalCollection extends PromisedMongoCollection {
    * @param {Object} options : aggregation options
    * @return {Promise}
    */
-  _aggregateWrapper(restriction, date, pipeline, options) {
+  _aggregateWrapper(restriction, date, pipeline, options, cursor = false) {
```
Comment (yangchristian) [inline lib/classes/TemporalCollection.js]: Can you add the new param to the doc block?

**Comment by yangchristian** [comment]
:+1: looks good, just that one minor comment. Thanks!

File: lib/classes/TemporalCollection.js (hunk 1 above)
Comment (bsouthga) [inline lib/classes/TemporalCollection.js]: good catch, fixed

**Comment by mcgrit** [comment]
Looks good to me.

**Pull request merged**